### ISSUE TITLE
Disqualification: Navigator tried to step on the queen's toes.

### ISSUE THREAD

**Issue by GNiklasch** [GNiklasch/VinceAnt#6]
After an encounter with the Monorail gang, we had seen fit to spawn a second navigator - and then one of the two ended up in a cell diagonally adjacent to the queen's, instead of laterally, and tried to correct this by stepping onto the queen's cell.
```
+---+---+---+---+---+---+---+
|   |   |   |   |   |   |   |
| w | y | g | b | p | y | w |
+---+---+---+---+---+---+---+
|   |   |V1 |   |   |   |   |
| w | k | g | b | p | y | w |
+---+---+---+---+---+---+---+
|   |   |   |   |   |VQ |   |
| w | y | g | b | y | y | w |
+---+---+---+---+---+---+---+
|   |   |   |   |V1 |   |   |
| w | y | g | b | g | y | w |
+---+---+---+---+---+---+---+
|   |   |   |   |   |   |   |
| w | y | g | b | p | y | w |
+---+---+---+---+---+---+---+
```

**Comment by GNiklasch** [comment]
(This does not need backporting to Windmill; it resulted from an oversight in porting the Windmill lightspeed-mode code to VinceAnt. In the original, the secretary's lightspeed strategy function was only called after checking that the queen is really laterally adjacent, not diagonally. I had mistakenly edited out this check when adapting the Windmill `ANT_STAFF` code to VinceAnt's `ANT_NAVIGATOR`, without adding a replacement elsewhere.)

**Comment by GNiklasch** [comment]
Fixed in master by 2cb75dc. Will be in the next release (no timeline yet).

**Comment by GNiklasch** [comment]
(Admittedly the fix is ugly and could do with some streamlining...)
